pkg/restapi/webhook/readme_webhook.txt

```diff
@@ -0,0 +1,5 @@
+Aries-agentd uses a webhook mechanism to communicate events back to the controller.
```

Comment:
Please use markdown format not txt files.

Reply:
Add it under docs and in the same file with aries-agentd executable docs

Reply:
Will do. I will also close this and add it to my other PR as suggested by @fqutishat.